pkg/controller/namespace/namespace_controller.go

```diff
@@ -79,11 +71,7 @@ func NewNamespaceController(
 
 	// create the controller so we can inject the enqueue function
 	namespaceController := &NamespaceController{
-		kubeClient:                 kubeClient,
-		clientPool:                 clientPool,
```

Comment:
I'd be in favor of eliminating the extra layer if its not used elsewhere; I didn't see other uses.  The current construction doesn't seem to be used for mocking.  I can't sort out why we have it.